src/Service/AlertManager.php

```diff
@@ -263,12 +263,13 @@ public function formatAlert(NodeInterface $alert) {
 
     return [
       'title' => $alert->getTitle(),
-      'textColor' => $alert->field_alert_text_color->entity->field_color->value,
-      'bgColor' => $alert->field_alert_color->entity->field_color->value,
-      'description' => $alert->field_alert_description->value,
+      'textColor' => $alert?->field_alert_text_color?->entity?->field_color?->value,
+      'bgColor' => $alert?->field_alert_color?->entity?->field_color?->value,
+      'alertStyle' => $alert?->field_alert_style?->value,
+      'description' => $alert?->field_alert_description?->value,
       'iconColor' => $iconColor,
       'linkUrl' => $url,
-      'linkText' => $alert->field_alert_link->title,
+      'linkText' => $alert?->field_alert_link?->title,
```

Comment:
what would happen when no title set? How empty link would be rendered?

Reply:
@podarok the link wouldn't display https://github.com/open-y-subprojects/openy_node_alert/blob/c8bb70042fb55fe1862334440d9bc434255690e8/js/openy-alerts/src/components/HeaderAlertItem/HeaderAlertItem.js#L124